core/platforms/firewalld/bareos-director.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>Bareos Director daemon (bareos-dir)</short>
```

Comment:
```suggestion
  <short>Bareos Director (bareos-dir)</short>
```
In https://docs.bareos.org/DocumentationStyleGuide/CommonNames.html we define a common writing style for the daemon.
Still, there are places where the naming has been "Bareos Director Daemon" /with captital D). I'd also agree to tihs.

Reply:
```suggestion
  <short>Bareos Director Daemon (bareos-dir)</short>
```

Reply:
I will prefer to have the 3 named Daemon.